libs/community/langchain_community/vectorstores/vdms.py

```diff
@@ -171,586 +129,664 @@ class VDMS(VectorStore):
             )
     """
 
+    @override
     def __init__(
         self,
         client: vdms.vdms,
         *,
-        embedding: Optional[Embeddings] = None,
-        collection_name: str = DEFAULT_COLLECTION_NAME,  # DescriptorSet name
-        distance_strategy: DISTANCE_METRICS = "L2",
+        embedding: Embeddings,
```

Comment:
breaking change

Reply:
@baskaryan, this change wasn't necessary. I reverted it